#Request

bool request

---

#Response

trajectory_msgs/JointTrajectory prbt_trajectory
